### PR TITLE
Feature/scenes

### PR DESCRIPTION
**Pull request by ryanlaycock**
Waiting on https://github.com/whip-ma-whop-ma-studios/space-game-poc/pull/6 to be merged, then this can be updated (no merge conflicts pls 🤞 ) and reviewed

### CODE REVIEW

File: Assets/SceneLoaders/RocketController.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        StartCoroutine(RunAnimation());
+    }
+
+    IEnumerator RunAnimation()
+    {
+        Debug.Log("Going up!");
+        _player.SetActive(false);
+        _animator.Play("RocketLiftOff");
+        yield return new WaitForSeconds(_transitionTime);
```
Comment (CalebDevonport) [inline Assets/SceneLoaders/RocketController.cs:30]: Should this transition time be the same one from the base class, or one private to this specific class? Isn't that referencing the circle fade animation time, not the rocket flying animation time?

**Comment by CalebDevonport** [comment]
Not sure how to comment on files so doing a PR level one. Unsure on the names for the interactable controllers, maybe could do with something indicating they're interactable controllers? Even if it's just like RedFlowerInteractableController, instead of RedFlowerController, it'll make it clearer? Just because stuff like RocketController makes it sound to me like a controller for flying/shooting a rocket whereas in reality it's an interaction to start an animation etc.

**Comment by CalebDevonport** [comment]
> Not sure on having the interactable objects, such as flowers, and the rocket etc, be named controllers. In my mind there's nothing really controlling them, it's more that you're interacting with them, I'm not sure on a more suitable name though.

**Review by CalebDevonport — APPROVED** [review]

File: Assets/SceneLoaders/RocketController.cs (hunk 1 above)
Comment (ryanlaycock) [reply Assets/SceneLoaders/RocketController.cs:30]: Ooo yeh good spot

**Comment by ryanlaycock** [comment]
> Not sure how to comment on files so doing a PR level one. Unsure on the names for the interactable controllers, maybe could do with something indicating they're interactable controllers? Even if it's just like RedFlowerInteractableController, instead of RedFlowerController, it'll make it clearer? Just because stuff like RocketController makes it sound to me like a controller for flying/shooting a rocket whereas in reality it's an interaction to start an animation etc.

Yeh I agree, we should come up with a better naming system. Merging for now though cause effort

**Pull request merged**